### PR TITLE
docs-infra: several fixes related to running docs examples with ivy

### PR DESCRIPTION
**Pull request by gkalpak**
Several fixes mainly related to running docs examples locally against Ivy.
See individual commits for details.

##
_Marking for `master-only`, because most affected files have diverged between master and patch and running against Ivy is mainly useful on master._

### CODE REVIEW

File: aio/tools/examples/example-boilerplate.js [hunk 1]
```diff
@@ -80,9 +80,7 @@ class ExampleBoilerPlate {
     }
 
     if (ivy) {
-      // We only need the "es2015" bundles as the CLI webpack build does not need
-      // any other formats for building and serving.
-      shelljs.exec(`yarn --cwd ${SHARED_PATH} ngcc --properties es2015`);
+      shelljs.exec(`yarn --cwd ${SHARED_PATH} ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points`);
```
Comment (petebacondarwin) [inline aio/tools/examples/example-boilerplate.js]: I don't understand why this fix is actually required.
If we are building via CLI then any missing formats will be compiled by CLI-ngcc integration as needed anyway.
If we are not using CLI then the `--create-ivy-entry-points` would not help. 

(There is a typo in the commit message: "is they have other format")

**Review by petebacondarwin — COMMENTED** [review]
@matsko this PR was merged without any reviews!!!

@gkalpak Lovely commit messages! Thanks.
One typo and confusion over the point of 1st commit.

File: aio/tools/examples/example-boilerplate.js (hunk 1 above)
Comment (gkalpak) [reply aio/tools/examples/example-boilerplate.js]: The problem is that `ngcc` will fail if it cannot process any format (from the specified `--properties`) for a package. The error looks like this:

```
$ /.../node_modules/.bin/ngcc --properties es2015
Error: Unable to process any formats for the following entry-points (tried es2015): 
  - /.../node_modules/angular-in-memory-web-api
```

I'll update the commit message with this info.

##
TBH, I am a little confused as to why this happened on one of Alex's PRs (and it not happening on master already), but I didn't want to spend much time on this at this point and aligning the command with what would run for new apps (and what would run through the cli) seemed like a good change anyway.

**Review by devversion — APPROVED** [review]
LGTM for dev-infra.

**Review by petebacondarwin — APPROVED** [review]

**Pull request closed without merging**